test.js

```diff
@@ -122,6 +122,10 @@ test('fetchFlight', async (t) => {
 			t.equal(typeof flight.arrivalGate, 'string')
 		}
 		if (flight.delay !== null) t.equal(typeof flight.delay, 'number')
+
+		if (flight.trail) {
```

Comment:
```suggestion
		if (flight.trail !== null) {
```

Reply:
Will await comment on above issue, before resolving this in the appropriate manner.